src/utils/utils_definitions.hpp

```diff
@@ -64,6 +64,24 @@ enum CreatureIcon_t {
 	CREATUREICON_HIGHERRECEIVEDDAMAGE = 1,
 	CREATUREICON_LOWERDEALTDAMAGE = 2,
 	CREATUREICON_TURNEDMELEE = 3,
+	CREATUREICON_GREENBALL = 4,
```

Comment:
there are 2 sets of these icons

first set is monster icons (1-3). It supports numbers but they're optional.
the other set is icons for all creatures (1-21). Numbers are always displayed here.

Second byte of the packet determines which set you're sending.
I've documented the packet structure in a gist.
See https://gist.github.com/Zbizu/476f86b647eb907495deacdfa8594906 for details.